vision/google/cloud/vision/image.py

```diff
@@ -85,28 +85,58 @@ def source(self):
         """
         return self._source
 
-    def _detect_annotation(self, feature):
+    def _detect_annotation(self, features):
         """Generic method for detecting a single annotation.
 
-        :type feature: :class:`~google.cloud.vision.feature.Feature`
-        :param feature: The ``Feature`` indication the type of annotation to
-                        perform.
+        :type features: list of:class:`~google.cloud.vision.feature.Feature`
```

Comment:
Just use `list` here and explain the type of each element in the list in the `param` doc